packages/ui5-application-inquirer/src/prompts/validators.ts

```diff
@@ -33,6 +33,10 @@ export async function validateFioriAppProjectFolder(targetDir: string): Promise<
     const appRoot = await findRootsForPath(targetDir);
     if (appRoot) {
         return t('validators.folderContainsFioriApp', { path: appRoot.appRoot });
+    } // Check targetDir is in CAP project sub directories && check if targetDir is a CAP project root directory
```

Comment:
This comment is superfluous and should be covered by the JSdoc of `checkPathForCapApp`

Reply:
Thank you, changed as requested.